src/main/java/io/singularitynet/MissionHandlers/VideoProducerImplementation.java

```diff
@@ -55,20 +55,16 @@ public int getWidth()
     @Override
     public int getHeight() { return this.videoParams.getHeight(); }
 
-    private AbstractMap.Entry<ByteBuffer, int[]> getRGBFrame()
+    private int[] getRGBFrame(ByteBuffer buffer)
     {
         Framebuffer framebuffer = MinecraftClient.getInstance().getFramebuffer();
         int i = framebuffer.textureWidth;
         int j = framebuffer.textureHeight;
-        ByteBuffer resBuffer = BufferUtils.createByteBuffer(i * j * 4);
-        resBuffer.flip();
-        GlStateManager._readPixels(0, 0, i, j, NativeImage.Format.RGBA.toGl(), GL11.GL_UNSIGNED_BYTE, resBuffer);
-        int[] sizes = new int[3];
+        GlStateManager._readPixels(0, 0, i, j, NativeImage.Format.RGBA.toGl(), GL11.GL_UNSIGNED_BYTE, buffer);
```

Comment:
We can try RGB instead of RGBA to save some bandwidth and  time

Reply:
As I've already said, using RGB ruining the game. Game just falls with error for some reason.